packages/desktop-client/src/components/tooltips.tsx

```diff
@@ -138,9 +184,10 @@ export class Tooltip extends Component {
     }
 
     const box = contentEl.getBoundingClientRect();
-    const anchorEl = this.target.parentNode;
+    const anchorEl = this.target.parentNode as HTMLElement;
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏same question here

Reply:
I think this is necessary because `.partentNode` is of type `ParentNode` which you can't call `getBoundingClientRect` on.

Reply:
Might need to add a [type guard](https://blog.logrocket.com/how-to-use-type-guards-typescript/) here. That would solve the problem.

In my experience manual type coercions tend to lead to subtle bugs that are very hard to catch. Hence why I'm so nitpicky about them. Sorry about that.